datalogistik/dataset.py

```diff
@@ -238,8 +252,11 @@ def get_one_table(self, table=None):
             )
         return all_tables[index]
 
-    # TODO: should these not return the specs, but the dataset itself?
-    def get_csv_dataset_spec(self, table):
+    def get_csv_dataset_spec(self, table: Table):
```

Comment:
```suggestion
    def get_csv_dataset_spec(self, table: Table) -> Tuple[pads.CsvFileFormat, dict]:
```
...not very sure about this

Reply:
Hm I think the schema will be a `pyarrow.schema`